# generated by patch-package 6.5.0 on 2022-03-31 20:04:26
#
# command:
#   npx patch-package @emotion/serialize --exclude nothing
#
# declared package:
#   @emotion/serialize: 1.0.3
#
diff --git a/node_modules/@emotion/serialize/package.json b/node_modules/@emotion/serialize/package.json
index ef39314..15044d3 100644
--- a/node_modules/@emotion/serialize/package.json
+++ b/node_modules/@emotion/serialize/package.json
@@ -27,9 +27,5 @@
     "src",
     "dist",
     "types/*.d.ts"
-  ],
-  "browser": {
-    "./dist/emotion-serialize.cjs.js": "./dist/emotion-serialize.browser.cjs.js",
-    "./dist/emotion-serialize.esm.js": "./dist/emotion-serialize.browser.esm.js"
-  }
+  ]
 }
